LoopKitUI/Views/TableViewTitleLabel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TableViewTitleLabel.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-01-17.
+//  Copyright © 2020 Tidepool Project. All rights reserved.
+//
+
+import UIKit
+
+public class TableViewTitleLabel: UILabel {
+    
+    override public init(frame: CGRect) {
+        super.init(frame: frame)
+        initFont()
+    }
+
+    required public init?(coder: NSCoder) {
+        super.init(coder: coder)
+        initFont()
+    }
+    
+    override public func awakeFromNib() {
+        super.awakeFromNib()
+        initFont()
+    }
+    
+    public func initFont() {
+        font = .loopTitleFontGroupedInset
```

Comment:
[nit] `self.` or not to `self.`, that is the question. :)

Reply:
me: Oh great complier...  
complier: `self` is needed in this case!!
:)